android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PhysicalWebBroadcastService.java

```diff
@@ -0,0 +1,205 @@
+/*
+HereYouGoService.java
```

Comment:
Put the normal copyright header here.  Move the description down just above the class definition.